zulia-server/src/main/java/io/zulia/server/index/ShardReader.java

```diff
@@ -82,14 +81,13 @@ public ShardReader(int shardNumber, DirectoryReader indexReader, DirectoryTaxono
 		this.zuliaPerFieldAnalyzer = zuliaPerFieldAnalyzer;
 		this.queryResultCache = Caffeine.newBuilder().maximumSize(indexConfig.getIndexSettings().getShardQueryCacheSize()).recordStats().build();
 		this.pinnedQueryResultCache = Caffeine.newBuilder().recordStats().build();
-		this.searchExecutor = Executors.newVirtualThreadPerTaskExecutor();
+
```

Comment:
weird this didn't format to remove the empty?

Reply:
Yeah weird, I removed it